src/components/sidebar/left-sidebar/AnalysisMethods.tsx

```diff
@@ -1,9 +1,8 @@
 import React from "react";
-import { MIN_DENOMINATOR } from "../../../metaAnalysis";
 import InformationText from "../../shared/InformationText";
 import './AnalysisMethods.css';
 
-
+const MIN_DENOMINATOR = 100
```

Comment:
do we still use this? if not get rid plz

Reply:
we do still use it apparently